eli5/base.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+import attr
+from typing import Dict, List, Tuple, Union
+
+
+@attr.s
+class Explanation(object):
```

Comment:
Could you please add docstrings to all these classes? They don't have to be long; maybe you can move some of comments from explain_.. functions here as well.